Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -244,14 +244,44 @@ theorem stabilizer_smul_eq_stabilizer_map_conj (g : G) (a : α) :
   rw [mem_stabilizer_iff, ← smul_left_cancel_iff g⁻¹, smul_smul, smul_smul, smul_smul,
     inv_mul_cancel, one_smul, ← mem_stabilizer_iff, Subgroup.mem_map_equiv, MulAut.conj_symm_apply]
 
+/-- The natural group equivalence between the stabilizers of two elements in the same orbit. -/
+def stabilizerEquivStabilizer {a b : α} {g : G} (hg : g • b = a) :
```

Comment:
Since you use `{a b : α} {g : G}` in almost all of the lemmas, and this makes up the `Stabilizer` section, perhaps adding `variable {a b : α} {g : G}` around line 238 would make sense.

Reply:
I did that, I also changed the order of these implicit arguments.